src/core_modules/capture-core/components/Pages/New/RegistrationDataEntry/RegistrationDataEntry.component.js

```diff
@@ -1,30 +1,74 @@
 // @flow
 import React from 'react';
+import i18n from '@dhis2/d2-i18n';
 import type { OwnProps } from './RegistrationDataEntry.types';
 import { EnrollmentRegistrationEntry, TeiRegistrationEntry } from '../../../DataEntries';
 import { scopeTypes } from '../../../../metaData';
 import { useScopeInfo } from '../../../../hooks/useScopeInfo';
+import { useRegistrationFormInfoForSelectedScope } from '../../../DataEntries/common/useRegistrationFormInfoForSelectedScope';
+import { InfoIconText } from '../../../InfoIconText';
+import { useCurrentOrgUnitInfo } from '../../../../hooks/useCurrentOrgUnitInfo';
+
+
+const translatedTextWithStylesForProgram = (trackedEntityName, programName, orgUnitName) =>
+    (<>
+        {i18n.t('Saving a {{trackedEntityName}} in', { trackedEntityName })} <b>{programName}</b>
+        {orgUnitName && <>{' '}{i18n.t('in')} <b>{orgUnitName}</b></>}.
+    </>);
+
+
+const translatedTextWithStylesForTei = (trackedEntityName, orgUnitName) =>
+    (<>
+        {i18n.t('Saving a {{trackedEntityName}}', { trackedEntityName })} <b>{i18n.t('without')}</b> {i18n.t('enrollment')}
+        {orgUnitName && <>{' '}{i18n.t('in')} <b>{orgUnitName}</b></>}.{' '}
+        {i18n.t('Enroll in a program by selecting a program from the top bar.')}
+    </>);
```

Comment:
This text if you see on the design has some bolds and some not bolds. This is why I am hacking it like this.

Reply:
I know I've used these bold parts in the new single event page, but I've later realised the concatenations will cause problems, at least for rtl languages. I suggest we keep it nonbold for now? @Bekkalizer , any insights?

Reply:
Does it make sense to make a bug ticket for this? Then I could merge this and we come back to this in isolation.